order_bot/request.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
```

Comment:
use strict on all files or non, not a mix of the two. 

I believe I've addressed this 2 times already. This 3rd is my final warning on this topic. I will simply reject the code next time.